src/randoop/reflection/DefaultReflectionPredicate.java

```diff
@@ -58,6 +57,18 @@ public boolean test(Class<?> c) {
     return visibility.isVisible (c);
   }
 
+  /**
+   * {@inheritDoc}
+   * Does checks for the following cases:
+   * <ul>
+   * <li>Main methods
```

Comment:
No, I don't see any reason not to call main. Though we might want to make sure that the timeout handling is better.